frontend/src/components/GetRandomQuestion.js

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from 'react';
+import DOMPurify from 'dompurify';
+import { getRoomById } from '../api/collaboration';
+import { fetchQuestionById } from '../api/questions';
+import styles from '../styles/components/GetRandomQuestion.module.css';
+
+const GetRandomQuestion = ({ roomId }) => {
+    const [question, setQuestion] = useState({ title: '', complexity: '', category: '' });
+
+    useEffect(() => {
+        const fetchQuestion = async () => {
+            const room = await getRoomById(roomId);
+            if (!room) {
+                return;
+            }
+
+            const question = await fetchQuestionById(room.questionId);
+            setQuestion(question);
+        };
+
+        fetchQuestion();
+    }, []);
+
+    return (
+        <div>
+            <h1>{question.title}</h1>
+            <h3>{question.complexity}</h3>
+            <h3>{question.category}</h3>
+            <div 
+                className={styles["content"]}
+                dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(question.description) }} 
```

Comment:
What does this do?

Reply:
it renders the question with the content from database. DOMPurify is just a layer of protection against XSS